Cabal/Distribution/Types/PackageDescription.hs

```diff
@@ -296,6 +299,18 @@ allBuildInfo pkg_descr = [ bi | lib <- allLibraries pkg_descr
   --FIXME: many of the places where this is used, we actually want to look at
   --       unbuildable bits too, probably need separate functions
 
+-- | As they appear on the cabal file, Paths_* module has no special treatment.
```

Comment:
@Paths_*@ (so that it gets Haddock formatted correctly). Also, the Haddock should say what the function does, not a general remark about how Paths modules get handled. It's important for a user to know `allModuleNames` applies to all NON-autogenerated modules, this is not obvious without looking at the source. Also this seems to miss LibV09? I'm not sure this function is the right approach. Look at my comment at its call site.